src/ui/controls/RadioListControl.tsx

```diff
@@ -40,6 +40,15 @@ export class RadioListControl extends React.Component<IStringControlProps, void>
   }
 
   /** @override */
+  componentDidUpdate() {
+    const { limitedToValues, selectedValue } = this.props.variable;
+    let index = limitedToValues.indexOf(selectedValue);
+    let id = `${CSS.RMX_RADIO_LIST_ITEM}-Theme-${index}`;
+    let materialRadio = this.refs[id]["MaterialRadio"];
+    materialRadio.check();
+  }
+
```

Comment:
The `this.refs[key]` syntax is deprecated:

https://facebook.github.io/react/docs/refs-and-the-dom.html

Please use the callback syntax instead.